lib/ui/text/paragraph_builder.cc

```diff
@@ -253,7 +256,8 @@ void ParagraphBuilder::pushStyle(tonic::Int32List& encoded,
       style.font_style = static_cast<txt::FontStyle>(encoded[tsFontStyleIndex]);
 
     if (mask & tsFontFamilyMask)
-      style.font_family = fontFamily;
+      style.font_families.push_back(fontFamily);// = fontFamily;
```

Comment:
Remove these comments

(If fontFamily is merged into fontFamilyFallback, then this block will no longer be needed)